community.html

```diff
@@ -43,13 +43,13 @@ <h2>Help others</h2>
   <h2>Help the project</h2>
   <ul>
     <li class="package"><h3>Package an App</h3>
-      <p>Check out our <a href="https://docs.sandstorm.io/en/latest/vagrant-spk/packaging-tutorial/">tutorial</a> to get started packaging apps for the <a href="https://apps.sandstorm.io/">App Market</a>. If you get stuck, <a href="https://groups.google.com/group/sandstorm-dev"> email us</a> for help or hop in our <a href="https://matrix.to/#/#sandstorm:libera.chat">Matrix room</a>.</p>
+      <p>Check out our <a href="https://docs.sandstorm.io/en/latest/vagrant-spk/packaging-tutorial/">tutorial</a> to get started packaging apps for the <a href="https://apps.sandstorm.io/">App Market</a>. If you get stuck, <a href="https://groups.io/g/sandstorm-dev-group">email the dev list</a> for help or hop in our <a href="https://matrix.to/#/#sandstorm:libera.chat">Matrix room</a>.</p>
     <li  class="feedback"><h3>Provide Feedback</h3>
       <p>To provide suggestions for each app: visit the <a href="https://apps.sandstorm.io/">Sandstorm App Market</a>, click on the app, click on the app's Code link, then navigate to its issue tracker. You can report bugs in Sandstorm itself on our <a href="https://github.com/sandstorm-io/sandstorm/issues">issue tracker</a>.</p>
     <li class="design"><h3>Design App Icons</h3>
-      <p>Many apps packaged for Sandstorm need icons and design help. If you have design skills and would love to help improve apps, <a href="https://groups.google.com/group/sandstorm-dev">send us an e-mail</a> and we'll connect you with apps in need.</p>
+      <p>Many apps packaged for Sandstorm need icons and design help. If you have design skills and would love to help improve apps, <a href="https://groups.io/g/sandstorm-dev-group">send us an e-mail</a> and we'll connect you with apps in need.</p>
     <li class="usability"><h3>Usability Test</h3>
-      <p>Usability tests are a great way to identify user problems and areas for improvement. Find volunteers, record them as they try Sandstorm, review the videos, then <a href="https://github.com/sandstorm-io/sandstorm/issues">file issues</a>. For more help read <a href="https://www.nngroup.com/articles/usability-101-introduction-to-usability/">Usability 101</a> by Jakob Nielsen and <a href="https://groups.google.com/group/sandstorm-dev">e-mail us</a> for tips.</p>
+      <p>Usability tests are a great way to identify user problems and areas for improvement. Find volunteers, record them as they try Sandstorm, review the videos, then <a href="https://github.com/sandstorm-io/sandstorm/issues">file issues</a>. For more help read <a href="https://www.nngroup.com/articles/usability-101-introduction-to-usability/">Usability 101</a> by Jakob Nielsen and <a href="https://groups.io/g/sandstorm-dev-group">e-mail us</a> for tips.</p>

```

Comment:
Do the landing pages give us any formatting flexibility? I wonder if we could have some headers and stuff to make it quicker to see which one the user is on.

Reply:
This one has "e-mail us" and the above you changed to "email the dev list". In case you wanted it uniform (and it's okay if not).

BTW nit: "e-mail" vs "email" maybe we could make that consistent.

Reply:
Honestly I think the right thing to do is to just eradicate this instance of the link because it's redundantly redundant, and we aren't in a position to support a ton of usability testing anyways.